qemu/tests/single_driver_install.py

```diff
@@ -1,11 +1,34 @@
-import os
 import re
 import logging
 
-from virttest import utils_misc
 from virttest import error_context
-from aexpect import ShellCmdError
-from aexpect import ShellProcessTerminatedError
+from virttest.utils_misc import set_winutils_letter
+from virttest.utils_windows import (virtio_win,
+                                    wmic)
+from virttest.utils_test.qemu import windrv_verify_running
+
+
+def _chk_cert(session, cert_path):
+    chk_cmd = "certutil -verify %s"
+    chk_cmd %= cert_path
+    out = session.cmd(chk_cmd)
+    if re.search("Expired certificate", out, re.I):
+        logging.warning("Certificate '%s' is expired!", cert_path)
+    if re.search("Incomplete certificate chain", out, re.I):
+        logging.warning("Incomplete certificate chain! Details:\n%s", out)
```

Comment:
I'm wondering why not raise error here?
Does the cert still could be added although it's incomplete or expired?

Reply:
I had checked this with @lijinlijin , she said that would be okay since not all windows guests would be suffered by the certification problem.